internal/oci/oci.go

```diff
@@ -188,6 +189,20 @@ func (r *Runtime) Timezone() string {
 	return r.config.Timezone
 }
 
+// GetContainerRuntimeMemory returns the runtime memory for a container
+// using the runtime handler configuration value (container_min_memory).
+func (r *Runtime) GetContainerRuntimeMemory(runtimeHandler string) (int64, error) {
+	rh, err := r.getRuntimeHandler(runtimeHandler)
+	if err != nil {
+		return int64(0), err
+	}
+	// we can skip error checking since the value was checked
+	// at the time of checking the runtime config
+	// nolint: errcheck
+	value, _ := units.RAMInBytes(rh.ContainerMinMemory)
```

Comment:
```suggestion
	// We can skip error checking since the value was checked
	// at the time of initializing the runtime handlers features.
	value, _ := units.RAMInBytes(rh.ContainerMinMemory) //nolint: errcheck

```